lib/config.js

```diff
@@ -66,6 +66,9 @@ var DEFAULTS = {
   logUncaughtExceptions: false, // TODO: Change to `true` in the v4.0.0
```

Comment:
Everything in this file is just configuration handling (including converting the wildcards into regular expressions)